samples/SqlExtensionSamples/OutputBindingSamples/QueueTriggerProducts.cs

```diff
@@ -5,22 +5,22 @@
 using Microsoft.Extensions.Logging;
 using static SqlExtensionSamples.ProductUtilities;
 
-namespace SqlExtensionSamples.OutputBindingSamples
+namespace SqlExtensionSamples
```

Comment:
Why did you remove the namespace?

Reply:
Removed for consistency with the other sample functions, none of the other ones have it